.github/workflows/prbot.yml

```diff
@@ -1,18 +1,157 @@
 name: PR Robot
-on:
-- pull_request
-
+on: [pull_request]
 jobs:
-  pr-robot:
+  build:
+    name: "Build"
     runs-on: ubuntu-latest
     env:
       GITHUB_CREDENTIALS: ${{ secrets.GITHUB_TOKEN }}
     steps:
-    - uses: actions/checkout@v2
-      with:
+      - uses: actions/checkout@v2
+        with:
           fetch-depth: 0
-    - uses: actions/setup-node@v1
-      with:
-          node-version: '14'
-    - run: npm i
-    - run: npm run pr:report
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+      - run: |
+          npm ci
+          npx lerna bootstrap
+      - run: npm run build
+      - run: |
+          tar -czvf build.tar.gz \
+            packages/bueno/dist \
+            packages/headless/dist \
+            packages/auth/dist \
+            packages/atomic/dist \
+            packages/atomic/docs \
+            packages/atomic/headless \
+            packages/atomic/loader \
+            packages/atomic/storybook-static \
+            packages/atomic/www
+      - uses: actions/upload-artifact@v2
+        with:
+          path: build.tar.gz
+          retention-days: 1
+      - run: npm run pr:report
```

Comment:
This step could be moved to a separate job. However, it only takes about 4 seconds to run.